xclim/utils.py

```diff
@@ -54,10 +54,15 @@
 # TODO: The pint library does not have a generic Unit or Quantity type at the moment. Using "Any" as a stand-in.
 
 units = pint.UnitRegistry(autoconvert_offset_to_baseunit=True)
+
 units.define(
-    pint.unit.UnitDefinition("percent", "%", (), pint.converters.ScaleConverter(0.01))
+    pint.unit.UnitDefinition(
+        "percent", "%", ("pct",), pint.converters.ScaleConverter(0.01)
+    )
 )
 
+units.define("1 = fraction = [] = frac")
```

Comment:
What is this doing?  Related to dimensionless case I think?

Reply:
Correct. Some netcdf files use units=1 for dimensionless quantities. This probably needs to be better tested however.

Reply:
In fact, this doesn't seem to work. Will remove for now.